### ISSUE TITLE
Extracting and saving mAP during training

### ISSUE THREAD

**Issue by LeoMilone98** [fizyr/keras-retinanet#1577]
Hi,
I am currently training a retina-net network with resnet50 backbone for object detection with my custom csv dataset. 

The code I'm using to train is the following:
!keras-retinanet/keras_retinanet/bin/train.py --backbone resnet50 --weights /path/to/PreTrainedWeights.h5 --batch-size {batch_size} --epochs {epochs} --steps {steps_per_epoch} --lr {lr} --snapshot-path /path/to/snapshot --compute-val-loss csv  /path/to/Annotation.csv  /path/to/Classes.csv --val-annotations  /path/to/ValAnnotation.csv

I woulk like to know if during training it is possible to save the mAP value and the respective epoch in a matrix. Thanks
